tests/ignite/metrics/test_accuracy.py

```diff
@@ -62,57 +62,50 @@ def test_binary_wrong_inputs():
         acc.update((torch.randint(0, 2, size=(10,)).long(), torch.randint(0, 2, size=(10, 5, 6)).long()))
 
 
-def test_binary_input():
-
+@pytest.fixture(params=[item for item in range(12)])
```

Comment:
Can't we just use range instead of list ?
```suggestion
@pytest.fixture(params=range(12))
```